### ISSUE TITLE
Dash.js

### ISSUE THREAD

**Issue by hazre** [moshisushi/hlsjs-ipfs-loader#30]
Wrong place to ask but I was wondering if there is something like this but for Dash.js

**Comment by vaultec81** [comment]
I don't believe so to my knowledge, but if you do find a project that is doing dash.js please let me know!
